packages/gatsby/index.d.ts

```diff
@@ -34,6 +34,8 @@ export {
 
 export * from "gatsby-script"
 
+export { IGatsbyResolverContext } from "./src/schema/type-definitions"
```

Comment:
I would guess we can skip this and I overlooked an already exported type for https://github.com/gatsbyjs/gatsby/pull/38805/files#diff-2a342b3e927ced0bfb4d19fe7f56ef73e0f08094f83526aaccc252d29af11870R44

🤔

Reply:
We don't really have good types for schema customization - most of it is ... `any` 🙈